packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -8,8 +8,8 @@
 %global gem_name katello
 %global prereleasesource pre.master
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global mainver 3.18.0
-%global release 5
+%global mainver 4.0.0
```

Comment:
Please also update foreman_{min,max}_version in this file

Reply:
Updated. Thanks for the catch